resteasy-jaxrs/src/main/java/org/jboss/resteasy/spi/AsyncClientResponseProvider.java

```diff
@@ -0,0 +1,14 @@
+package org.jboss.resteasy.spi;
+
+import java.util.concurrent.CompletionStage;
+
+/**
+ * ??????????????????????????????/
+ * @author rsigal
```

Comment:
A simple sentence instead of the "???" ? :)

Reply:
@asoldano, that was supposed to remind me to add some javadoc. #fail